--- a/usr/lib/clang/22/include/cuda_wrappers/new
+++ b/usr/lib/clang/22/include/cuda_wrappers/new
@@ -98,9 +98,9 @@
 __device__ inline void *operator new[](__SIZE_TYPE__, void *__ptr) CUDA_NOEXCEPT {
   return __ptr;
 }
-#endif
 __device__ inline void operator delete(void *, void *) CUDA_NOEXCEPT {}
 __device__ inline void operator delete[](void *, void *) CUDA_NOEXCEPT {}
+#endif

 #pragma pop_macro("CUDA_NOEXCEPT")

---
